Makefile

```
@@ -197,17 +199,17 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
```

```suggestion
	cd config/profile/$(PROFILE) && $(KUSTOMIZE) edit set image controller=$(IMG)
```

I think the controller image should be set in the right profile. Otherwise, I noticed that we lost the ability to deploy a specific image for the operator.
For example, I use to test PRs using for example: `make deploy IMG=quay.io/rhdh-community/operator:0.3.0-pr-62-e0e3b4d`.
The operator deployed should then use the image specified. But with the changes here, it would continue to use the default image (`quay.io/rhdh-community/operator:0.3.0`).

This makes me wonder if the `config/manager` folder is needed at all and should not be removed. WDYT?

---

config/manager should be removed, yes